Playbook.yaml

```diff
@@ -0,0 +1,84 @@
+- name:  App - Java with Maven - Ubuntu Linux
+  hosts: servers
+  become: yes
+  remote_user: user
+  gather_facts: no
+  
+  vars:
+    maven_major_version: 3
+    maven_version: 3.8.6
+  
+  tasks:
+    - name: Install Default Java JRE/JDK Pkg
+      apt:
+        pkg:
+        - default-jre
+        - default-jdk
+
+    - name: Create Maven Directory
+      file:
+        path: /usr/local/apache-maven
+        state: directory
+        mode: '0775'
+        recurse: yes
+        owner: root
+        group: root
+
+    - name: Download and Unarchive Maven
+      unarchive:
+        src: https://dlcdn.apache.org/maven/maven-{{ maven_major_version }}/{{ maven_version }}/binaries/apache-maven-{{ maven_version }}-bin.tar.gz
+        dest: /usr/local/apache-maven
+        remote_src: yes
+
+    - name: Config ~/.bashrc
+      blockinfile:
+        create: yes
+        path: ~/.bashrc #/root/.bashrc
```

Comment:
if this path means **/root/.bashrc** use the absolute path

Reply:
Suggestion has been applied.